src/main/java/delight/nashornsandbox/internal/JsSanitizer.java

```diff
@@ -36,6 +35,28 @@
  */
 @SuppressWarnings("restriction")
 public class JsSanitizer {
+
+	private static final Class<?> JDK_NASHORN_ScriptObjectMirror_CLASS;
+	private static final Class<?> STANDALONE_NASHORN_ScriptObjectMirror_CLASS;
+
+	static {
+		Class<?> tmp_JDK_NASHORN_ScriptObjectMirror_CLASS = null;
+		// TODO what behavior do we want here?
+		try {
+			tmp_JDK_NASHORN_ScriptObjectMirror_CLASS = Class.forName("jdk.nashorn.api.scripting.ScriptObjectMirror");
+		} catch (ClassNotFoundException e) {
+			System.out.println("JDK Nashorn not found");
```

Comment:
I assume in the final version we wouldn't want to log out here?

Reply:
Hello, like I wrote in the description, this is a basis for discussion and must not be merged as is.
I think there should be somewhere a log statement indicating which version of Nashorn is used and why, but that should probably be at debug level.